src/Model/ImageLoading.py

```diff
@@ -197,13 +203,23 @@ def missing_interop_elements(read_file) -> bool:
     """
     is_missing = []
     for onko_required_attribute in all_iods_required_attributes:
-        if (not hasattr(read_file, onko_required_attribute)
-            or read_file[onko_required_attribute].is_empty):
+        if (
+            not hasattr(read_file, onko_required_attribute)
+            or read_file[onko_required_attribute].is_empty
+        ):
+            if onko_required_attribute == "StudyID":
```

Comment:
Change in logic to make Onko more forgiving with respect to StudyID.
I can revert this change if it's not desirable (I just keep running in to it on data I have from NCIA).